### PR TITLE
Development

### PR DESCRIPTION
**Pull request by viditjain5598**
Adds:
- Ability to CRUD interviews
- Ability to CRUD users
- Ability to attach files to interviews
- Ability to send reminder mails to interviewees

TODO:
- Manage Authentication 
- Error Detection

## Listed view of all interviews
image

## New Interview
image

## Interview Tab
image

### CODE REVIEW

File: app/controllers/interviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  def new
+    @interview = current_user.interviews.build
+  end
+
+  def create
+    @interview = current_user.interviews.build(interview_params)
+
+    if @interview.save
+      redirect_to @interview, notice: "Successfully saved interview details"
+      UserMailer.with(:interview=>@interview, :user_id=>member).updation_mails.deliver_later(wait_until: @interview.scheduled_time - 30.minutes)
```
Comment (abhishek-8) [inline app/controllers/interviews_controller.rb:21]: member ?

Comment (viditjain5598) [reply app/controllers/interviews_controller.rb]: Fixed in 6f4aeb8

**Comment by ayu-111** [comment]
Please add these functionalities:
[ ] Add multiple users in a same interview.
[ ] Add resumes
[ ] Checks for clashing interview
[ ] Reminder for Interviews after time is edited.
[ ] Add resume links in the mailers

**Comment by viditjain5598** [comment]
@ayu-111 The adding resume and reminder when interview timings features are updated are already present. I'll add the rest soon.

**Pull request merged**